staging/src/k8s.io/apiserver/pkg/server/hooks.go

```diff
@@ -49,11 +49,6 @@ type PreShutdownHookFunc func() error
 type PostStartHookContext struct {
 	// LoopbackClientConfig is a config for a privileged loopback connection to the API server
 	LoopbackClientConfig *restclient.Config
-	// StopCh is the channel that will be closed when the server stops.
-	//
-	// Deprecated: use the PostStartHookContext itself instead, it contains a context that
-	// gets cancelled when the server stops. StopCh keeps getting provided for existing code.
-	StopCh <-chan struct{}
```

Comment:
when did we deprecate it? Longer ago?

Reply:
> Fully deprecate PostStartHookContext.StopCh

Fully remove it, no?

Reply:
Its deprecated ages go. I found this message and just took on myself to do the cleaning :)

Reply:
updated release notes

Reply:
updated git commit message too